.github/workflows/dbt_slim_ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           [[ -z "$test" ]] && { echo "Success: All models have a custom schema"; exit 0; } || { echo "Found models without custom schema:"; echo "$test"; exit 1; }
 
       - name: dbt seed
-        run: "dbt seed $PROFILE --select @state:modified --exclude tag:prod_exclude tag:remove --state ."
+        run: "dbt seed $PROFILE --select state:modified --exclude tag:prod_exclude tag:remove --state ."
```

Comment:
i think the `@` here runs any seed that is in a dbt lineage within context of PR, even if the specific model that calls the seed test isn't modified. removing it helps here, but we've also added for helping other PRs. i think rather than remove it, i'll look to fix the seed issue in another PR